copy.go

```diff
@@ -236,6 +237,7 @@ func structToStruct(filter FieldFilter, src, dst *reflect.Value, userOptions *op
 // options are used in StructToStruct and StructToMap functions to modify the copying behavior.
 type options struct {
 	DstTag string
+	SrcTag string // copy according to src tag, i.g. json
```

Comment:
Please, move this comment 1 line above.
Also add a comment for the `DstTag` field